openmc/universe.py

```diff
@@ -124,6 +125,110 @@ def name(self, name):
         else:
             self._name = ''
 
+    def find(self, point):
+        """Find cells/universes/lattices which contain a given point
+
+        Parameters
+        ----------
+        point : 3-tuple of float
+            Cartesian coordinatesof the point
+
+        Returns
+        -------
+        list
+            Sequence of universes, cells, and lattices which are traversed to
+            find the given point
+
+        """
+        p = np.asarray(point)
+        for cell in self._cells.values():
+            if p in cell:
+                if cell._type in ('normal', 'void'):
```

Comment:
Actually it's kind of awkward right now; we only have a fill_type property which is inconsistent with what's stored in _type, e.g., when _type is 'normal' fill_type is 'material'. Also, if the fill is void, fill_type returns None, so I found it cleaner to rely on _type here.

I would actually suggest that we make fill=None correspond to a void material, and that when you create a cell it is void by default so the user would never have to say fill='void'. What do you think? If you agree, I can implement it here, get rid of _type altogether, and just rely on fill_type.